modules/distillery.cpp

```diff
@@ -550,7 +550,9 @@ void LapH::distillery::add_to_perambulator(const size_t dil_t, const size_t dil_
     // checking if smeared or stochastic sink must be computed
     if (!param.dilution_type_si[nbs][1].compare("F")){ // smeared sink
       // running over sink time index
+#pragma omp for schedule(dynamic) nowait
       for(size_t t = 0; t < T; ++t){
+        //printf("add_to_perambulator thread: %d\n", omp_get_thread_num());
         if( param.use_zgemm ) {
         // ZGEMM MULTIPLICATION 
```

Comment:
note that the ZGEMM part is completely broken now, we should really just remove it... but until the status of the sLapH toolset is clarified, I certainly don't want to invest more time than necessary